dragnet/features.py

```diff
@@ -97,47 +97,47 @@ class CSSFeatures(object):
-                              'top',
-                              'content',
-                              'link',
-                              'title',
-                              'comment',
-                              'tools',
-                              'rating',
-                              'ss'],
-                     'class':['menu',
-                              'widget',
-                              'nav',
-                              'share',
-                              'facebook',
-                              'cat',
-                              'top',
-                              'content',
-                              'item',
-                              'twitter',
-                              'button',
-                              'title',
-                              'header',
-                              'ss',
-                              'post',
-                              'comment',
-                              'meta',
-                              'alt',
-                              'time',
-                              'depth',
-                              'thread',
-                              'author',
-                              'tools',
-                              'reply'
-                              'url',
-                              'avatar',
-                              'ss']}
+    attribute_tokens = {'id': ['nav',
+                               'ss',
+                               'top',
+                               'content',
+                               'link',
+                               'title',
+                               'comment',
+                               'tools',
+                               'rating',
+                               'ss'],
+                        'class': ['menu',
+                                  'widget',
+                                  'nav',
+                                  'share',
+                                  'facebook',
+                                  'cat',
+                                  'top',
+                                  'content',
+                                  'item',
+                                  'twitter',
+                                  'button',
+                                  'title',
+                                  'header',
+                                  'ss',
+                                  'post',
+                                  'comment',
+                                  'meta',
+                                  'alt',
+                                  'time',
+                                  'depth',
+                                  'thread',
+                                  'author',
+                                  'tools',
+                                  'reply',
+                                  'url',
+                                  'avatar']  # , 'ss']  <-- duplicate?!
```

Comment:
So, this is interesting. I was getting a _different_ failure before I commented out the duplicate css tag. The error arises in `ContentExtractionModel.make_features_from_blocks()`, where there's a mismatch in numpy array shapes:

```
Traceback (most recent call last):
  File "/Users/burtondewilde/Desktop/github/dragnet/test/test_models.py", line 38, in test_models
    content = m.analyze(self._html)
  File "/Users/burtondewilde/Desktop/github/dragnet/dragnet/content_extraction_model.py", line 78, in analyze
    return self.analyze_from_blocks(blocks_, return_blocks=blocks)
  File "/Users/burtondewilde/Desktop/github/dragnet/dragnet/content_extraction_model.py", line 82, in analyze_from_blocks
    features = self.make_features_from_blocks(blocks)
  File "/Users/burtondewilde/Desktop/github/dragnet/dragnet/content_extraction_model.py", line 107, in make_features_from_blocks
    features[:, offset:offset_end] = f(blocks, train)
ValueError: could not broadcast input array from shape (222,37) into shape (222,36)
```